sanic/request.py

```diff
@@ -1,7 +1,6 @@
 import asyncio
 import email.utils
 import json
```

Comment:
minor tweak, we don't need to import json if ujson doesn't exist, we should make this import in the except branch

Reply:
Good catch!